extractor/src/test/resources/damls/EnumMod.daml

```diff
@@ -0,0 +1,33 @@
+-- Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+-- Port of daml-lf/encoder/src/test/lf/Enum_all_.lf
+
+module EnumMod where
+
+import Daml.Script
+
+data Color = Red | Green | Blue deriving (Show, Eq)
+
+template Box
+  with
+    x: Color
+    party: Party
+  where
+    signatory party
+
+createContracts =
+  script do
+    party <- allocatePartyWithHint "EnumMod" (PartyIdHint "EnumMod")
```

Comment:
I relied on hints, which is sub-optimal but should be good enough for the time being. Ideally extractor will be removed before we need to move to Canton.